File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.cs
using System;
public class Solution 
{
    public int[] solution(long n) 
    {
        string s = n.ToString();
        int[] ans = new int[s.Length];
        for(int i = 0; i < s.Length; i++)
        {
            ans[i] = s[s.Length - 1 - i] - '0';
        }
        return ans;
    }
}

File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(long long n) 
{
    string strN = to_string(n);
    vector<int> vec;
    while(!strN.empty())
    {
        vec.push_back(strN.back() - '0');
        strN.pop_back();
    }
    
    return vec;
}